tf-modules/archive/clean_executions.tf

```diff
@@ -1,6 +1,7 @@
 resource "aws_sqs_queue" "clean_executions_dead_letter_queue" {
   name                       = "${var.prefix}-cleanExecutionsDeadLetterQueue"
   receive_wait_time_seconds  = 20
+  message_retention_seconds  = 1209600
```

Comment:
Updating these to match the max value to make this the same as it was in the kes implementation